### DISCUSSION TITLE
What does all the key shorteings mean?

### DISCUSSION THREAD

**Discussion by dennisfransen** [nickcoutsos/keymap-editor#107] [Q&A] [answered]
Quite new to the custom keyboard community and i'm trying to figure out what, for example &kp, &to, &tog means.
I'm guessing &kp means **key press**.

But i can't figure out what all the others mean. And my goal here is to make a key toggle some behavior.
So my ask here is if someone can tell me what all the shorteings mean?

Maybe it could be added in the select menu in the UI?

**Comment by nickcoutsos** [comment, marked as answer]
This tool seeks to simplify some of the manual work in modifying keymaps but, at this time at least, expects a level of familiarity with ZMK. Behaviors are all described in detail in the ZMK documentation; here's [Key Toggle](https://zmk.dev/docs/behaviors/key-toggle).

I see that the behavior selection list isn't consistent about using full names or descriptions, I can review my data and make sure that's all filled out wherever it's available.

Edit: I've updated the behavior selection list to include the `description` that gets parsed from ZMK's behavior yaml files. I might look at adding links to the relevant behavior page in ZMK's documentation but there would be some manual work there in terms of collecting URLs and deciding how best to present it in the UI.
